src/routes/device-logs.ts

```diff
@@ -307,14 +323,22 @@ function handleStreamingWrite(
 			// Don't flush if the backend is reporting as unavailable
 			if (buffer.length && backend.available) {
 				// Even if the connection was closed, still flush the buffer
-				const promise = backend.publish(ctx, buffer);
+				const publishingToRedis = backend.publish(ctx, buffer);
+				const publishingToLoki = shouldPublishToLoki()
+					? loki
+							.publish(ctx, buffer)
+							.catch((err) =>
+								captureException(err, 'Failed to publish logs to Loki'),
+							)
+					: Promise.resolve();
```

Comment:
```suggestion
					: undefined;
```

Reply:
Updated, do you prefer `undefined` for speed?